tests/test_import.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import absolute_import
+
+
+# What follows is the actual test;
+# because import * is only allowed on module level, this cannot be run inside a
+# proper test. If the import fails, the test runner will complain loudly.
+
+from attr import *
```

Comment:
If you really wanted to I believe you could put this in another file (not called test_) and then have a test that imports *that* file.  :)

Reply:
Yeah I did consider (and discarded) that idea initially, implemented now in e242b2d